### PR TITLE
revisions v2

### PR DESCRIPTION
**Pull request by drwelby**
PR for comments @m-mohr @pjhartzell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,21 +18,74 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
...
-| roles           | `[string]`       | see [https://github.com/radiantearth/stac-spec/pull/989] |
+| offset          | `[integer]`    | **REQUIRED** Offset to first bit in the field |
+| length          | `[integer]`    | **REQUIRED** Number of bits in the field |
+| classes         | `[Class]`      | **REQUIRED** Classes represented by the field values |
+| roles           | `[string]`       | see [Asset Roles](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#asset-roles) |
 | description     | `string`       | A short description of the classification. |
 | name           | `string`             | Short name of the class for machine readibility, optional |
 
+
+A Bit Field stores classes within a range of bits in a data value. The range is described by the offest of the first bit from the rightmost position, and the length of bits used to store the class values.
```
Comment (pjhartzell) [inline README.md:44]: minor typo: change offest to offset

**Review by pjhartzell — CHANGES_REQUESTED** [review]
README revisions and additions look good! I noted one minor typo.

**Comment by pjhartzell** [comment]
With respect to number #9, should we update `value` in the Class Object to something like `number|[number]|Range Object`? 

Range Object being the same as in the Collection specification: https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#range-object

**Comment by drwelby** [comment]
I am horribly inconsistent about using periods in lists so I will get the formatting standardized at some point.

**Comment by drwelby** [comment]
closing, see latest PR

**Pull request closed without merging**